### ISSUE TITLE
Generation start errors in Google Colab

### ISSUE THREAD

**Issue by s-ledyakhov** [arenasys/qDiffusion#6]
Hello
I noticed that you had an update, everything worked well in Colaba half a day ago, and now an error is crashing.
Do you have any suggestions on how this can be fixed?
Thanks for your work

```
SERVER Traceback (most recent call last):
  File "/content/sd-inference-server/server.py", line 108, in run
    self.wrapper.img2img()
  File "/usr/local/lib/python3.10/dist-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/content/sd-inference-server/wrapper.py", line 752, in img2img
    metadata = self.get_metadata("img2img",  width, height, batch_size, self.prompt, seeds, subseeds)
  File "/content/sd-inference-server/wrapper.py", line 466, in get_metadata
    m["strength"] = format_float(self.strength)
  File "/content/sd-inference-server/wrapper.py", line 92, in format_float
    return f"{x:.4f}".rstrip('0').rstrip('.')
ValueError: Unknown format code 'f' for object of type 'str'
```

**Comment by arenasys** [comment]
this should have been fixed by c2c57cb632e8362e40c7092cc4cdec6e93125edf. update and remake the colab and see if the error persists

**Comment by s-ledyakhov** [comment]
> this should have been fixed by [c2c57cb](https://github.com/arenasys/qDiffusion/commit/c2c57cb632e8362e40c7092cc4cdec6e93125edf). update and remake the colab and see if the error persists

Faced some errors while updating, but everything worked out
Thank you for your project!
